Assets/AppCenter/Plugins/AppCenterSDK/Core/UWP/AppCenterInternal.cs

```diff
@@ -207,6 +207,8 @@ private static void Prepare()
 
                     /**
                      * Workaround for known IL2CPP issue.
+                     * Currently it is not required, but since this workaround stores settings in separate file, 
+                     * and not generic unity mechanism, a migration is required to transition users who store data in old file to built-in mechanism. 
```

Comment:
I don't see the todo in the feature work items by the way, are we tracking that somewhere?

Reply:
It was agreed to keep this without changes